graphql/src/main/java/com/scalar/db/graphql/schema/TableGraphQlModel.java

```diff
@@ -65,7 +64,6 @@ public TableGraphQlModel(String namespaceName, String tableName, TableMetadata t
     this.tableName = Objects.requireNonNull(tableName);
     this.tableMetadata = Objects.requireNonNull(tableMetadata);
 
-    this.transactionEnabled = ConsensusCommitUtils.isTransactionalTableMetadata(tableMetadata);
     this.fieldNames =
         ConsensusCommitUtils.removeTransactionalMetaColumns(tableMetadata).getColumnNames();
```

Comment:
It works with the config `scalar.db.transaction_manager=jdbc` and a normal table schema without transactional metadata.
In the graphql module, `ConsensusCommitUtils` is only used [once](https://github.com/scalar-labs/scalardb/blob/2ac7f46c5ef582fb91818c78147f1cf50d261dee/graphql/src/main/java/com/scalar/db/graphql/schema/TableGraphQlModel.java#L68) to remove the transactional metadata from a table for listing the fields that are not metadata.

Reply:
Let me clarify one more thing.
So, if a user uses JDBC transaction and manages a table that has some column named `tx_id` (something used in ConsensusCommit), then it's going to be an issue?

Reply:
Ah, yes. Columns prefixed with `tx_` like `tx_id` won't appear in the GraphQL schema since they will be removed by `ConsensusCommitUtils.removeTransactionalMetaColumns` even if the JDBC native transaction is used.

Reply:
Is it going to be fixed in another PR? If so, I will just approve it.

Reply:
Ah, it may be more complex than I thought.

First of all, we have two implementations for the transaction mode, `Consensus Commit` and `JDBC transaction`. And I think we should be able to access the `Consensus Commit` transactional tables through both storage and transaction modes.

And In storage mode and transaction mode with `JDBC transaction`, we should treat the `Consensus Commit` transactional columns as normal columns, but in transaction mode with `Consensus Commit`, we should treat them as special ones and hide them from users.

Reply:
Thank you for the comment.

In my understanding, we should hide columns like the Consensus Commit meta columns (`tx_*`) from GraphQL users only when the table is accessed via the Consensus Commit transactional mode. I think that the problem is whether we can know if the Consensus Commit transaction mode is going to be used when generating the GraphQL schema. (The GraphQL schema is created from the information of `TableMetadata` only which is retrieved from `DistributedStorageAdmin`.)

To determine if the meta columns should be hidden from the schema or not, one way might be to check each table with `ConsensusCommitUtils.isTransactionalTableMetadata`. The meta columns are hidden if and only if the table has all meta columns.
Otherwise, I think we might have to expose all columns without checking if they are meta columns or not.
What do you think?

Anyway, this is a problem of the schema generating phase and is not directly related to this PR. I will work on this in another PR.

Reply:
Sure. Let's discuss it in another PR.